### ISSUE TITLE
pretrained model suppurt?

### ISSUE THREAD

**Issue by liyupeng-ing** [caiyuanhao1998/RSN#10]
Hi, nice work! Could you share the pretrained model about the RSN?

**Comment by Tiago-Roxo** [comment]
Hi. I am not one of the authors, but I can provide you my trained model here: https://drive.google.com/drive/folders/1atyNvjT_cIXq3_objomu6IJHm4K8CmET?usp=sharing. 
In there, you can find weights, the config file used, and the COCO score for the validation set (2017). I used the folder 4xRSN18 (in exps folders) and got AP 0.724. The model was trained for 1.15 million iterations.

**Comment by samsara177** [comment]
> Hi. I am not one of the authors, but I can provide you my trained model here: https://drive.google.com/drive/folders/1atyNvjT_cIXq3_objomu6IJHm4K8CmET?usp=sharing.
> In there, you can find weights, the config file used, and the COCO score for the validation set (2017). I used the folder 4xRSN18 (in exps folders) and got AP 0.724. The model was trained for 1.15 million iterations.

Hello, do you have a trained RSN18 model?

**Comment by ghost** [comment]
> Hi. I am not one of the authors, but I can provide you my trained model here: https://drive.google.com/drive/folders/1atyNvjT_cIXq3_objomu6IJHm4K8CmET?usp=sharing.
> In there, you can find weights, the config file used, and the COCO score for the validation set (2017). I used the folder 4xRSN18 (in exps folders) and got AP 0.724. The model was trained for 1.15 million iterations.

there is something wrong with the website you provided. Could you please provide it again? Thank you very much。
大佬，能把模型的链接再发一下吗？感激不尽！

**Comment by ghost** [comment]
> Hi. I am not one of the authors, but I can provide you my trained model here: https://drive.google.com/drive/folders/1atyNvjT_cIXq3_objomu6IJHm4K8CmET?usp=sharing.
> In there, you can find weights, the config file used, and the COCO score for the validation set (2017). I used the folder 4xRSN18 (in exps folders) and got AP 0.724. The model was trained for 1.15 million iterations.

the link is invalid now. Could you share it again?

**Comment by samsara177** [comment]
不好意思，我已经换工作了，目前没有这个模型的参数文件了。




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "caiyuanhao1998/RSN"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年7月28日(星期三) 下午4:58
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [caiyuanhao1998/RSN] pretrained model suppurt? (#10)





  
Hi. I am not one of the authors, but I can provide you my trained model here: https://drive.google.com/drive/folders/1atyNvjT_cIXq3_objomu6IJHm4K8CmET?usp=sharing.
 In there, you can find weights, the config file used, and the COCO score for the validation set (2017). I used the folder 4xRSN18 (in exps folders) and got AP 0.724. The model was trained for 1.15 million iterations.
  
there is something wrong with the website you provided. Could you please provide it again? Thank you very much。
 大佬，能把模型的链接再发一下吗？感激不尽！

**Comment by lofyol** [comment]
> Hi. I am not one of the authors, but I can provide you my trained model here: https://drive.google.com/drive/folders/1atyNvjT_cIXq3_objomu6IJHm4K8CmET?usp=sharing. In there, you can find weights, the config file used, and the COCO score for the validation set (2017). I used the folder 4xRSN18 (in exps folders) and got AP 0.724. The model was trained for 1.15 million iterations.

Hello, can you provide it again

**Comment by orestis-z** [comment]
> Hi. I am not one of the authors, but I can provide you my trained model here: https://drive.google.com/drive/folders/1atyNvjT_cIXq3_objomu6IJHm4K8CmET?usp=sharing. In there, you can find weights, the config file used, and the COCO score for the validation set (2017). I used the folder 4xRSN18 (in exps folders) and got AP 0.724. The model was trained for 1.15 million iterations.

Hello @Tiago-Roxo, @wql20000111, @yes-ai-do, and @samsara177
Does any of you have the weights above backed up and can share them with us?

**Comment by caiyuanhao1998** [comment]
Hi, here is the pre-trained model zoo:

https://mmpose.readthedocs.io/en/latest/model_zoo_papers/backbones.html#rsn-eccv-2020
